var/lib/istio/README.md

```diff
@@ -0,0 +1,13 @@
+This is the base of all istio configs.
```

Comment:
is a `var` top level folder the right place for this to live?

Reply:
It's most convenient for running istiod from the IDE :-)

It can go anywhere really - what I care about is preserving the structure that will be in the 
docker image so we don't have too much complexity.

Reply:
Is preserving the structure the right move or a common pattern? Why not just have a `config/` directory in the code and copy it to `var/lib/istio` in the docker file, that does not seem too complex

Reply:
Right now to make it easy to develop and debug, the code is looking for ./var/lib/istio - the docker image sets working dir as "/", so it works well in k8s or .deb.

When you run the app in IDE - will run in the base of the repo, and will also find the files. 

I can copy - and make the code more complicated to find it. It's not a problem to copy files in different directories in docker - the question is why ? We have files all over the place that go
in hard to predict locations - with var/lib/istio it is very clear what goes where, and we can skip all complexity, including for the .deb file.